components/pages/submission/Environmental/NewSubmissions/DropZone.tsx

```diff
@@ -55,8 +61,15 @@ const DropZone = ({
 		accept: acceptedExtensionsString,
 		disabled,
 		onDrop: useCallback(
-			(acceptedFiles: File[]) =>
-				acceptedFiles.forEach(validator(validationState, validationDispatch)),
+			(acceptedFiles: SubmissionFile[]) => {
+				// Accept files to corresponding array of files based on it's extension
+				acceptedFiles.forEach(validator(validationState, validationDispatch));
+
+				// Process sequencing files (i.e. tar.xz) and compute MD5
+				acceptedFiles
+					.filter((f) => getFileExtension(f.name) === acceptedFileExtensions.TAR_XZ)
```

Comment:
best practice standards strongly encourage avoiding single letter variable names
```suggestion
					.filter((file) => getFileExtension(file.name) === acceptedFileExtensions.TAR_XZ)
```